operations/helm/charts/mimir-distributed/templates/kafka/kafka-statefulset.yaml

```diff
@@ -0,0 +1,178 @@
+{{- if and .Values.kafka.enabled (not .Values.federation_frontend.disableOtherComponents) -}}
+{{- with .Values.kafka -}}
+{{- $replicas := .replicas | default 1 -}}
```

Comment:
Do we want to hard code the number of replicas as `1` and only support a single broker? I'm not sure if that would be a bad user experience since they'd be stuck with a very non-production setup but it seems like it would make the setup with Kafka simpler.

Reply:
I'd try to not overthink it and waited until real users came with their requests or questions.